File: assets/Practice/practice02/skeleton/mile_to_km.py
# mile_to_km.py
# Converts distance in miles to kilometers.

import sys

# main function
def main():
	KMS_PER_MILE = 1.609
	miles = float(raw_input("Enter distance in miles: "))
	kms = KMS_PER_MILE * miles

	print "That equals %9.2f km." % kms

# Runs the main method
if __name__ == "__main__":
	main()
	sys.exit(0)

File: assets/Practice/practice02/skeleton/mile_to_km.c
// Converts distance in miles to kilometers.
#include <stdio.h>
#define KMS_PER_MILE 1.609

int main(void) {
	double miles, kms;

	printf("Enter distance in miles: ");
	scanf("%lf", &miles);

	kms = KMS_PER_MILE * miles;
	printf("That equals %9.2f km.\n", kms);

	return 0;
}

